Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -265,6 +268,7 @@ private extension LoginListViewController {
         selectionButtonHeightConstraint?.update(offset: UIConstants.ToolbarHeight)
         self.view.layoutIfNeeded()
         tableView.setEditing(true, animated: true)
+        tableView.reloadData()
```

Comment:
This is necessary to show/hide the settings buttons when in search mode. They are hidden by setting the row height to zero.